### DISCUSSION TITLE
Clarification on Licensing for Music Created with LMMS

### DISCUSSION THREAD

**Discussion by codaishin** [LMMS/lmms#7637] [General]
Hello LMMS Team,

First of all, thank you for your incredible work on LMMS! 

I have a question regarding the licensing implications of music created with LMMS. While I understand the GPLv2 license applies to the LMMS software itself, I would like clarification on whether it extends to the music I compose using LMMS.

Specifically:
1. If I compose original music entirely on my own using LMMS, is the resulting music considered a derivative work of the software under GPLv2?
2. Can I freely distribute this music under other licenses, such as MIT or Apache 2.0, for use in an open-source version of a game?
3. Would there be any restrictions or licensing obligations if the music is included in a proprietary version of the game or in promotional material?

I deeply appreciate any guidance you can provide on this topic. My goal is to ensure that I respect LMMS’s license while proceeding with my project.

Thank you for your time and support!

[Text created with ChatGPT]

**Comment by qnebra** [comment]
Your text is owned by ChatGPT because you use it to create your text or not?

**Comment by qnebra** [comment]
Answering your question in more serious way, in case of you are an actual human being who use tool to ask serious question in nice way.

Music you create in LMMS is yours and your only. Developers just provide tool, in which you can create new piece of music. So, assuming you want to distribute music made in LMMS in your game, do not worry.

**Reply by codaishin** [reply]
Thank you. I did assume so. However, when it comes to legal stuff and licenses, I have come to not trust my gut. I have been wrong with how licenses worked in the past, so I figured I ask the repo owners themselves. And to make my question make sense, I used ChatGPT to turn my ramblings into fluent English :D.

Also thanks for considering I was human (another one fooled^^).
